### DISCUSSION TITLE
ollama support

### DISCUSSION THREAD

**Discussion by morandalex** [dbeaver/dbeaver#29282] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I think is interesting testing local LLM for sql generation 

**Describe the solution you'd like**
I should be able to put in settings the ollama endpoint so I can create query or fix query with ai helper using local llm

for more information about ollama check https://ollama.com/

**Comment by Destrolaric** [comment, marked as answer]
@morandalex The ability to work with LLAMA was already added for DBeaver Pro editions. it will be available for commercial editions in the 24.1.0 release.

**Reply by vanting** [reply]
I am using 25.0.0 release but still not seeing the Service option other than OpenAI.

**Reply by Destrolaric** [reply]
Are using Community edition, OLLama is not part of the CE version.

**Reply by morandalex** [reply]
yes, I am talking about CE edition!
Can you vote on this issue to enable it ?
https://github.com/dbeaver/dbeaver/issues/36951
